js/bitmap-map.js

```diff
@@ -14,6 +14,30 @@ window.openrailwaymap = {
 		"standard": "Infrastructure",
 		"maxspeed": "Maxspeeds",
 		"signals": "Signalling",
+	},
```

Comment:
Please squash this into the previous commit (or even earlier, depends on which introduced the actual bug).

Reply:
What is wrong here?